$MirOS: ports/emulators/qemu/patches/patch-block-vmdk_c,v 1.2 2005/11/15 12:25:08 tg Exp $
--- block-vmdk.c.orig	Tue Nov 15 11:52:05 2005
+++ block-vmdk.c	Tue Nov 15 11:52:33 2005
@@ -436,4 +436,6 @@ BlockDriver bdrv_vmdk = {
     vmdk_close,
     vmdk_create,
     vmdk_is_allocated,
+    NULL,
+    NULL
 };
